cogeo_mosaic/backends/dynamodb.py

```diff
@@ -296,27 +295,25 @@ def clean(self):
         """clean MosaicID from dynamoDB Table."""
         logger.debug(f"Deleting items for mosaic {self.mosaic_name}...")
 
-        # get items
+        # get quadkeys
         resp = self.table.query(
-            KeyConditionExpression="#mosaicId = :mosaicId",
-            ExpressionAttributeNames={"#mosaicId": "mosaicId", "#quadKey": "quadKey"},
-            ExpressionAttributeValues={":mosaicId": {"S": self.mosaic_name}},
-            ProjectionExpression="#quadKey",
+            KeyConditionExpression=Key("mosaicId").eq(self.mosaic_name),
+            ProjectionExpression="quadkey",
```

Comment:
Can you reuse `self._quadkeys` here?

Reply:
no because in _quadkeys we remove `-1`, we could append it 🤷‍♂️

Reply:
just feel that we will save 2 lines to add one back
```python
qks = self._quadkeys + ["-1"]
```

Reply:
I think that would be simpler than copying the dynamodb query twice. It also might be a good idea to have a dummy `_meta_quadkey` function:

```py
@property
def _meta_quadkey(self):
	return "-1"
```

then you could use
```py
quadkeys = [*self._quadkeys + self._meta_quadkey]
```

There are probably some other places where `_meta_quadkey` would help for clarity.